service/history/historyEngine.go

```diff
@@ -2463,6 +2463,13 @@ func validateActivityScheduleAttributes(attributes *workflow.ScheduleActivityTas
 		return &workflow.BadRequestError{Message: "A valid timeout may not be negative."}
 	}
 
+	if attributes.GetScheduleToCloseTimeoutSeconds() > wfTimeout ||
```

Comment:
Rather than failing decision tasks can we reduce the timeout to be the same as workflow timeout?
I'm worried this might cause issues in production.